clients/ts/signalr/tests/ServerSentEventsTransport.test.ts

```diff
@@ -33,12 +33,12 @@ describe("ServerSentEventsTransport", () => {
 
             await TestEventSource.eventSource.openSet;
 
-            expect(connectComplete).toEqual(false);
+            expect(connectComplete).toBe(false);
```

Comment:
To be, or not to be: that is the question: Whether 'tis nobler in the mind to suffer The slings and arrows of outrageous fortune, Or to take arms against a sea of troubles, And by opposing end them?